src/utils/validators.js

```diff
@@ -0,0 +1,5 @@
+import validator from 'validator';
+
+export function isNumber(value) {
+  return Number.isInteger(value) || (value && validator.isNumeric(value));
```

Comment:
why do you need `validator` here? isNaN should suffice no?

Reply:
This is the code from the web-client project (this code was added because a pr comment)